style.css

```diff
@@ -389,3 +393,187 @@ button:disabled {
   border: 1px solid #c1c7d0;
   color: #5e6c84;
 }
+
+@media only screen and (min-width: 768px) {
+  .humbarger {
+    display: none;
+  }
+
+  #navbar-menue {
+    display: inline-block;
+  }
+
+  #navbar-menue ul {
+    list-style-type: none;
+    display: flex;
+    gap: 32px;
+  }
+
+  #navbar-menue ul li a {
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  #phone {
+    height: 72px;
+    padding-left: 155px;
+    padding-right: 154px;
+  }
+
+  #bg {
+    height: 846px;
+    padding-left: 34.6px;
+    padding-right: 8px;
+    padding-top: 49px;
+  }
+
+  #bio {
+    height: 721px;
+    padding-left: 225px;
+    padding-right: 252px;
+    padding-top: 241px;
+    background-image: url('/images/Header-bg-desktop.png');
```

Comment:
- Hey, the image is not showing remotely but shows locally. Let's fix the file path to `./images/Header-bg-desktop.png`